modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -135,3 +136,54 @@ test('LPD-28908 update user information', async ({
 			)
 	).toBeVisible();
 });
+
+test('LPD-30589 Add Organization Team', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	editOrganizationPage,
+	page,
+	siteConfigurationDetailsPage,
+	siteSettingsPage,
+	teamsPage,
+	usersAndOrganizationsPage,
+}) => {
+	const organization = await apiHelpers.headlessAdminUser.postOrganization();
+	await usersAndOrganizationsPage.goToOrganizations();
```

Comment:
do we need to go to the page here (before the creation of the data) or we can create everything and then move in the page?